### ISSUE TITLE
1.21.4 support

### ISSUE THREAD

**Issue by ghost** [MCRcortex/nvidium#268]

**Comment by HemishK480** [comment]
when she gets time

**Comment by HemishK480** [comment]
between today and year 3102

**Comment by HemishK480** [comment]
so pretty soon

**Comment by mharbb** [comment]
> so pretty soon

yeah lol it hasn't been updated in like 3 months and 2 or 3 Minecraft updates.

**Comment by Skywalker8510** [comment]
Cortex has just been really busy lately so the mod will be updated it's just going to take some time because real life takes precedence over a mod that she's making for free.

**Comment by HemishK480** [comment]
it would be cool to have a bigger team similar to sodium/caffienemc

**Comment by A-r-s-a-n-e** [comment]
when there is no 1.21.2 even.. and you want a version that is way newer. Good luck waiting

**Comment by mharbb** [comment]
> when there is no 1.21.2 even.. and you want a version that is way newer. Good luck waiting

1.21.2 should've come a long time ago.

**Comment by KarmaMarazu** [comment]
> 1.21.2 should've come a long time ago.

just wait till cortex has time cortex is very busy and the mod will be updated when its updated till then ull just have to wait

**Comment by Samalando** [comment]
a mod is never early nor late, it arrives precisely when it needs to

**Comment by kierkat10** [comment]
would be nice to see 1.21.3 support, as cortex seems to have stopped working on the mod.
though, they can do whatever.

**Comment by KarmaMarazu** [comment]
> would be nice to see 1.21.3 support, as cortex seems to have stopped working on the mod. though, they can do whatever.

updates will happen when they happen the mod is not forgotten about these things take time as well as cortex currently working on rewriting a different mod of hers voxy

**Comment by kierkat10** [comment]
yes, i know that cortex is currently not working on the mod, i'm just commenting that my game runs very poorly without nvidium, and so it would be better if the mod was actually updated for more recent versions of the game. i don't want to force them into doing it.
i will check out this voxy mod though, maybe it could be cool.

**Comment by Skywalker8510** [comment]
i you really want the latest version of Minecraft you can run a server on your local machine on the latest version with viaversion and viabackward then play with an older client using nvidium.

**Comment by kierkat10** [comment]
interesting, i may look at that.

**Comment by Felix14-v2** [comment]
That's unrelated. Sodium has its own translucency sorting implementation by Douira.
Also
![image](https://github.com/user-attachments/assets/eba3d732-b012-425f-88d2-e181f859c12d)

**Comment by DemonQueenIshino** [comment]
> That's unrelated. Sodium has its own translucency sorting implementation by Douira. Also ![image](https://github.com/user-attachments/assets/eba3d732-b012-425f-88d2-e181f859c12d)

that screenshot doesn’t even hold up anymore as AMD GPUs RDNA2 and newer have had openGL mesh shader support since may 2024.

**Comment by KarmaMarazu** [comment]
> that screenshot doesn’t even hold up anymore as AMD GPUs RDNA2 and newer have had openGL mesh shader support since may 2024.

well no not exactly its not in the public full release drivers yet so you are still waiting on it to actually work
soon it will be

**Comment by Orangenal-name** [comment]
> > when there is no 1.21.2 even.. and you want a version that is way newer. Good luck waiting
> 
> 1.21.2 should've come a long time ago.

brooooo nobody uses 1.21.2 cause it was only out for like two days when 1.21.3 dropped, likely that if 1.21.2 is supported it's just cause it's not very different from 1.21.3

**Comment by Samalando** [comment]
i dont think thats the point they are going for here

**Comment by Lavendarbunny** [comment]
I hope nvidium wasn't abandoned

**Comment by KarmaMarazu** [comment]
> I hope nvidium wasn't abandoned

Dw it hasnt been yet cortex is just working on other things for now (see voxy rewrite) and personal life things. It will get here when it gets here but its not been forgotten

**Comment by SCPRedMage** [comment]
You may be interested in [this pull request](https://github.com/MCRcortex/nvidium/pull/282); I've been using this version for a few days on 1.21.4 with Sodium 0.6.9.

**Comment by KarmaMarazu** [comment]
https://github.com/drouarb/nvidium/releases and the 1.21.4 branch to compile it urself
